test/externalTests.sh

```diff
@@ -59,10 +59,13 @@ function test_truffle
       for d in node_modules node_modules/truffle/node_modules
       do
       (
-        cd $d
-        rm -rf solc
-        git clone --depth 1 -b v0.5.0 https://github.com/ethereum/solc-js.git solc
-        cp "$SOLJSON" solc/
+        if [ -d "$cd" ]
```

Comment:
`$cd`?

Reply:
Damn, I'm tired, was supposed to be `$d`.